packages/npm-package-json-lint-config/index.js

```diff
@@ -57,7 +57,10 @@ const defaultConfig = {
 				'main',
 				'module',
```

Comment:
[`exports` is a standard field we could also include in this ordering:](https://nodejs.org/api/packages.html#exports)

```suggestion
				'module',
				'exports',
```